content/SCALE/SCALEUIReference/TopToolbar/Alerts/AlertSettingsScreen.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Alert Settings Screens
+title: Alert Settings Screen
 weight: 10
 tags:
 - scalealerts
```

Comment:
add - scaleclustering to the tags list, 

You can add other tags if you think there are other areas mentioned that could benefit by adding the tag.